src/material/list/list.ts

```diff
@@ -90,4 +91,12 @@ export class MatListItem extends MatListItemBase {
   ) {
     super(element, ngZone, listBase, platform, globalRippleOptions, animationMode);
   }
+
+  /**
+   * Return true if this item is the current active page in a `<mat-nav-list>`. This method is
+   * safe to use with server-side rendering.
+   */
+  _ssrSafeIsCurrentPage() {
+    return this._hostElement.nodeName === 'A' && this._activated;
```

Comment:
I would write this as a getter instead so we don't have any logic in the data binding:

```
_getAriaCurrent(): string | null {
  return this._hostElement.nodeName === 'A' && this._activated ? 'page' : null;
}
```